portfolio/src/main/webapp/index.html

```diff
@@ -6,9 +6,12 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComment()">
```

Comment:
This function doesn't seem to be defined anywhere.  Did you mean to use getRandomMessage?